### PR TITLE
Implemented features for manage campaign

### PR DESCRIPTION
**Pull request by gblend**
Feature: Manage Campaign

Manage Campaign features for the fundmylaptop to make campaign management functional by:

·      Allowing Investee create new campaigns
·      End a particular campaign
·      Share campaigns on facebook, twitter, whatsapp, linkedin etc
·      Allow investee to edit campaigns
·      Fetch recent involvements by an investee
.      List user campaigns

User_Manage_Campaign

A_signle_user_campaign_request_listings

User_Create_Campaign

User_Edit_Campaign

Share_on_twitter

Share_on_whatsapp

Share_on_fb

Social_Media_Share

### CODE REVIEW

File: database/migrations/2020_07_16_052445_add_request_id_to_requests.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public function up()
+    {
+        Schema::table('requests', function (Blueprint $table) {
+            //
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
```
Comment (onifs10) [inline database/migrations/2020_07_16_052445_add_request_id_to_requests.php:23]: what is this migration for

Comment (trustidkid) [reply database/migrations/2020_07_16_052445_add_request_id_to_requests.php:23]: Was initially intended to add **request_id** to **requests table**

Comment (trustidkid) [reply database/migrations/2020_07_16_052445_add_request_id_to_requests.php:23]: Was initially intended to add to **request_id** to **requests** table. The migration was later rollback. @onifs10

**Comment by trustidkid** [comment]
image

**Comment by trustidkid** [comment]
image

File: database/migrations/2020_07_16_052445_add_request_id_to_requests.php (hunk 1 above)
Comment (gblend) [reply database/migrations/2020_07_16_052445_add_request_id_to_requests.php:23]: All have been added as you requested @onifs10

**Pull request closed without merging**